mu-plugins/blocks/local-navigation-bar/postcss/style.pcss

```diff
@@ -14,7 +14,7 @@
 }
 
 .wp-block-wporg-local-navigation-bar {
-	height: var(--wp--custom--local-navigation-bar--spacing--height, 60px);
```

Comment:
I originally thought this already meant that it could be overridden by settings in the theme.json, but it turns out it can't? Asking this because I saw you explicitly set up 60px below.

Reply:
Good question; I've been following Kelly's example [here](https://github.com/WordPress/wporg-mu-plugins/blob/trunk/mu-plugins/blocks/query-filter/postcss/style.pcss#L325).

I can't remember exactly why I had to make this change, but I think I found that JS wasn't reading the custom property default value correctly, probably because this var is scoped to the component and not on body like they are in other components.